docs/api/requests/withBearerToken.md

```diff
@@ -0,0 +1,45 @@
+---
+tags:
+  - authentication
+  - bearer auth
+---
+
+# withBearerToken
+
+Specifies bearer authentication.
+
+## Syntax
+
+```js
+withBearerToken(token)
+```
+
+## Usage
+
+### ✅  Correct Usage
+
+```js 
+await spec()
+  .get('/api/users')
+  .withBearerToken('token')
+  .expectStatus(200);
+```
+
+## Arguments
```

Comment:
In the new doc pages we no longer have the **arguments** section.  Please refer [withQueryParams](https://pactumjs.github.io/api/requests/withQueryParams.html#syntax) page for more details.